documentation/Extensions.md

```diff
@@ -102,6 +103,9 @@ Extensions, deletions, and legacy features supported by default
 * Specific intrinsics AMAX0, AMAX1, AMIN0, AMIN1, DMAX1, DMIN1, MAX0, MAX1,
   MIN0, and MIN1 accept more argument types than specified. They are replaced by
   the related generics followed by conversions to the specified result types.
+* When a scalar CHARACTER actual argument of the same kind is known to
+  have a length shorter than the associated dummy argument, it is extended
+  on the right with blanks, similar to assignment.
```

Comment:
It does imply a temporary, but I have not yet extended `ActualArgument` with the annotations that I plan to use to represent special processing of arguments before & after calls.  There will not be a copy back in these cases because the actual argument will no longer be a variable.